models/br_bd_diretorios_mundo/code/download_sh4_ncm.ipynb

```
@@ -2,9 +2,18 @@
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/luana/pipelines/.venv/lib/python3.10/site-packages/requests/__init__.py:102: RequestsDependencyWarning: urllib3 (1.26.20) or chardet (5.2.0)/charset_normalizer (2.0.9) doesn't match a supported version!\n",
+      "  warnings.warn(\"urllib3 ({}) or chardet ({})/charset_normalizer ({}) doesn't match a supported \"\n"
+     ]
+    }
+   ],
```

Ruff (0.15.13)

[error] Probable use of `requests` call without timeout

(S113)